### PR TITLE
adding yokwejuste as a contributor

### PR DESCRIPTION
**Pull request by yokwejuste**
## New pull request from [yokwejuste](https://github.com/yokwejuste)
- Adding yokwejuste as a contributor to osscameroon-blog.

### CODE REVIEW

File: content/post/contributors/yokwejuste.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Yokwejuste"
+date: 2022-01-19T14:53:19+01:00
+images: []
+categories: ["Web development", "Data Science", "Blockchain", "Cloud Computing"]
+tags: ["Open-source", "github"]
+authors: ["yokwejuste"]
+---
+Hi, I'm Yokwejuste, a young inspired blogger, aspiring to be a computer engineering. I like trying everything and being involved in everything tech 😅.
```
Comment (Sanix-Darker) [inline content/post/contributors/yokwejuste.md:9]:
```suggestion
Hi, I'm Yokwejuste, a young inspired blogger, aspiring to be a computer engineering. I like trying everything and being involved in tech 😅.
```

File: content/post/contributors/yokwejuste.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+The fields in which I'm involved include: Data Science, Web Development, blockchain, Cloud Computing and AI.
```
Comment (Sanix-Darker) [inline content/post/contributors/yokwejuste.md:10]:
```suggestion
The fields in which I'm involved are: Data Science, Web Development, blockchain, Cloud Computing and AI.
```

File: content/post/contributors/yokwejuste.md (hunk 1 above)
Comment (Hawawou) [reply content/post/contributors/yokwejuste.md:9]: Being involved is correct 😃

File: content/post/contributors/yokwejuste.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Yokwejuste"
+date: 2022-01-19T14:53:19+01:00
+images: []
+categories: ["Web development", "Data Science", "Blockchain", "Cloud Computing"]
+tags: ["Open-source", "github"]
+authors: ["yokwejuste"]
+---
+Hi, I'm Yokwejuste, a young inspired blogger, aspiring to be a computer engineer. I like to try everything and being involve in tech 😅.
+The fields in which I'm involved are: Data Science, Web Development, blockchain, Cloud Computing and AI.
```
Comment (RMPR) [inline content/post/contributors/yokwejuste.md:10]: Minor nitpick, why is "blockchain" lower case when everything else is upper case?

File: content/post/contributors/yokwejuste.md (hunk 2 above)
Comment (yokwejuste) [reply content/post/contributors/yokwejuste.md]: That's true, grammatical error, but still the same sense I guess!

**Pull request merged**